src/main/java/lotto/domain/Rank.java

```diff
@@ -18,11 +19,16 @@ public enum Rank {
         this.prize = prize;
     }
 
-    public static Rank rankOf(int matchingCount){
+    public static Rank rankOf(int matchingCount, boolean containsBonus){
+        if (matchingCount == 5 && containsBonus == true) {
```

Comment:
워낙 이전 구조가 좋아서 최소수정으로 요구사항 잘 반영해주셨습니다 😄 
`containsBonus`자체가 `boolean`이니 `== true`는 없어도 될 것 같아요 😄 
그리고 5라는 숫자는 매직넘버이니 좀 더 구체적이면 좋을 것 같습니다 🙏 
```suggestion
        if (matchingCount == THIRD.matchingCount && containsBonus) {
```